dev/ci/cases/pr/C96_atm3DVar_extended.yaml

```diff
@@ -23,3 +21,13 @@ skip_ci_on_hosts:
   - orion
   - hercules
   - awsepicglobalworkflow
+
+workflow:
+  engine: rocoto
+  rocoto:
+    maxtries: 2
+    cyclethrottle: 3
+    taskthrottle: 25
```

Comment:
Would it make sense to increase the `taskthrottle` for the extended test since it launches so many small jobs?  Maybe up to 75?